api/src/main/java/io/opencensus/metrics/MetricRegistry.java

```diff
@@ -92,5 +128,29 @@ public DoubleGauge addDoubleGauge(
           Utils.checkNotNull(unit, "unit"),
           labelKeys);
     }
+
+    @Override
+    public DerivedLongGauge addDerivedLongGauge(
+        String name, String description, String unit, List<LabelKey> labelKeys) {
+      Utils.checkListElementNotNull(
+          Utils.checkNotNull(labelKeys, "labelKeys"), "labelKey element should not be null.");
+      return DerivedLongGauge.newNoopDerivedLongGauge(
+          Utils.checkNotNull(name, "name"),
+          Utils.checkNotNull(description, "description"),
+          Utils.checkNotNull(unit, "unit"),
+          labelKeys);
+    }
+
+    @Override
+    public DerivedDoubleGauge addDerivedDoubleGauge(
+        String name, String description, String unit, List<LabelKey> labelKeys) {
+      Utils.checkListElementNotNull(
+          Utils.checkNotNull(labelKeys, "labelKeys"), "labelKey element should not be null.");
```

Comment:
s/"labelKey element should not be null."/"labelKey" 

same everywhere.

Reply:
I will fix this in separate PR, I see many places we did like this.